build/root/WORKSPACE

```diff
@@ -58,6 +58,14 @@ http_file(
     urls = mirror("https://storage.googleapis.com/kubernetes-release/network-plugins/cni-plugins-amd64-v0.6.0.tgz"),
 )
 
+CRI_TOOLS_VERSION = "1.0.0-beta.1"
```

Comment:
ug, when it the release?

Reply:
@Random-Liu @feiskyer ^

Reply:
Any chance we can sync this with the `cri_tools_version` rule?

Reply:
that would be ideal...i don't know enough about skylark tbh. Would you be cool if we tracked this with a ticket @luxas ?

Reply:
I agree (in principal) with @luxas - I'd rather this be specified only in one place.

I don't think there's any way for another rule to depend on this constant here in the WORKSPACE file, but we could create a `build/workspace.bzl` file, put `CRI_TOOLS_VERSION` in it, and then depend on that and in the `k8s_deb` rule.

(`ETCD_VERSION` should probably be treated similarly.)

Reply:
1.0.0-beta.1 is released on May 4. We're planning to release v1.11.0 for k8s v1.11